xl2times/transforms.py

```diff
@@ -2511,7 +2511,7 @@ def apply_transform_tables(
             )
 
             if not any(rows_to_update):
-                logger.info(f"A {Tag.tfm_mig.value} row generated no records.")
+                logger.warning(f"A {Tag.tfm_mig.value} row generated no records.")
```

Comment:
I think this should stay as info rather than a warning.

Reply:
After having another look at how the other messages are classified, I guess it is more consistent to have it as a warning for now.